tests/accounts/api/test_auth_login.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import json
+import jwt
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from tests.factories import UserFactory
+
+PASSWORD = "SuperPasswordSecret4"
+EMAIL = "hi@geeks.cat"
+DATA_POST = json.dumps({"email": EMAIL, "password": PASSWORD})
+LOGIN_URL = "/api/login/"
+CONTENT_TYPE_JSON = "application/json"
+
+@pytest.fixture
+def create_user() -> UserFactory:
+    user = UserFactory(email=EMAIL, is_active=True)
+    user.set_password(PASSWORD)
+    user.save()
+    return user
+
+
+@pytest.fixture
```

Comment:
I would like to add this fixture to `conftest.py` file in the root of tests folder, then all the tests will be able to use it, not only the tests in this file.